res/values/strings.xml

```diff
@@ -26,7 +26,7 @@
     <string name="hide_password">Hide password</string>
     <string name="not_now">Not now</string>
     <string name="never">Never</string>
-    <string name="one_moment">One moment…</string>
+    <string name="one_moment">One moment...</string>
```

Comment:
"…" is the unicode character 'HORIZONTAL ELLIPSIS'; following most recommendations and style guides, this should be favored over the three dots afaik.